wiki/Beatmapping/Beat_snapping/en.md

```diff
@@ -0,0 +1,22 @@
+---
+tags:
+  - snaps
+  - beat snap
+  - snapping
+---
+
+# Beat snapping
+
+::: Infobox
+![](img/beat_snap.png "The symbol for beat snap as portrayed in an early osu! editor build (circa 2007)")
+:::
```

Comment:
shouldn't this be in a trivia section and not at the top of the page?

Reply:
maybe it could be taken out of the article entirely -- I have only used it because it was there from the start. the editor doesn't have a dedicated _button_ for the beat snap anymore anyway, so the icon is all that remains